src/reducers/popover.tsx

```diff
@@ -0,0 +1,53 @@
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+export type PopoverIdType = 'COMMENTS_SUMMARY' | 'COMPARE_VERSIONS';
+
+export type PopoverState = {
+  current: PopoverIdType | undefined;
```

Comment:
Yeah, this was my intention. I wanted to use this to add a second popover button to the navbar. When clicking, it should close any other open navbar popover. I can't envision a future where we'd want it to work a different way but if we did then I guess we could move back to the controlled `Popover` and make a wrapper like `ControlledPopover` for the Redux version. I didn't do that because YAGNI 🤷‍♂